assets/dev/js/editor/ui-states/context-menu-mode.js

```diff
@@ -0,0 +1,24 @@
+import UiStateBase from 'elementor-api/core/states/ui-state-base';
+
+export class ContextMenuMode extends UiStateBase {
+	getId() {
+		return 'context-menu-mode';
+	}
+
+	getOptions() {
+		return {
+			on: '',
```

Comment:
Just as a good practice, maybe extract it to a constant?

```JS
const MODE_ON = 'on';
```

And then use this constant in every place where you set the UI state.

Reply:
Done

Reply:
Done